skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
I am also confused about your statement "I showed that it was not the case as that sample code worked for me."

Do you mean you use (1) your own repository to generate codes for sample.source.PKIX1 or (2) use my #143 to generate code for sample.source.PKIX1 ? 

Actually if you compare the generated code using your own master repository and the generated code using my #143, you can see they are different.

Please remember to execute **make regen** to re-generate these C files and H files, otherwise, the existed files will be the same. Of course you can delete them by hand.